testing/ncids-css-testing/stories/uswds-native/usa-combo-box/src/index.scss

```diff
@@ -1,5 +1,3 @@
 @use "styles/ncids";
 @forward "usa-label";
```

Comment:
This is just a question: why did these get removed?

Reply:
https://github.com/uswds/uswds/blob/e901bc69bd09adb07a7fefd0ff53751adf811bff/packages/usa-combo-box/_index.scss#L2

redundant. imported in the package.